code/_globalvars/lists/xenobiology.dm

```diff
@@ -0,0 +1,14 @@
+///These global lists exist to allow our element to have weight tables without having to be seperate instances.
+
+///Assoc list of cell line define | assoc list of datum | cell_line
+GLOBAL_LIST_INIT(cell_line_tables, list(

```

Comment:
GLOBAL_LIST_INIT_TYPED

Reply:
needs to be /list/list/

Reply:
wtf i thought i already made this init_typed, odd